README.md

```diff
@@ -109,6 +109,25 @@ You should see that the server is running by going to [http://localhost:4000](ht
 6. Go to the folder you installed srctweb and type jekyll serve -s './srctweb' -d './srctweb/_site'
 9. Navigate to localhost:4000 in a web browser
 
+#### MacOS
+
+1. Install homebrew if you have not already done it.
+2. Type "brew install rbenv ruby-build" to install rbenv, a tool to manage ruby versions.
+3. Run the command "rbenv init"
+    If you get an error such as:
```

Comment:
This isn't an error! The command `rbenv init` prints out instructions telling you what to do depending on what shell you're using.

Also for larger code snippets, surround them with three backticks (```) to get the nice Markdown code formatting.